### DISCUSSION TITLE
Welcome to Netflix-Prime-Auto-Skip Discussions!

### DISCUSSION THREAD

**Discussion by Dreamlinerm** [Dreamlinerm/Netflix-Prime-Auto-Skip#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ocuonline** [comment]
Hello
Your application is very nice and useful. But it freezes while watching amazon prime videos.

I use microsoft edge and firefox. I encountered the same problem in both.
I hope you can solve this problem with updates.

**Reply by Dreamlinerm** [reply]
which url are u using. I already tried to fix this problem for someone else, but apparently it is not fixed for you. You can join the discord to maybe help figure out what your problem is: https://discord.gg/5fBYathA7d

**Reply by ocuonline** [reply]
Thank you for your interest.

This is the url I used.

https://www.primevideo.com/region/eu

**Reply by Dreamlinerm** [reply]
Do have the newest version: v.1.1.45 or 44 installed?
In my country I can only access, amazon.com and amazon.de even with VPN since my acc is bound to them. Which country are you at. Can you test if you have the same problem on amazon.com, because it doesn't happen on amazon.de and i tried to fix it on amazon.com.

**Reply by ocuonline** [reply]
I live in Turkey. Unfortunately, I do not have access to Amazon.de or Amazon content.

Microsoft Edge version 44 is installed. (On Edge, the video starts freezing from the beginning and cannot be watched at all)
Firefox version 45 is installed. (After starting to watch on Firefox, it starts freezing and sound lags.)

When I use the application below on Edge, it works without any problems. But your application works very well on Netflix, your application is better. The problem only occurs on Amazon.

https://chromewebstore.google.com/detail/multi-skipper-skip-ads-in/bimlffinhbdhgpomhngmnhidjgnfcnoc

**Reply by Dreamlinerm** [reply]
hmm let me see how the extension you posted does it.

**Reply by Dreamlinerm** [reply]
When does the freezing happen? on the first ad. on every ad. or just normally watching?

**Reply by ocuonline** [reply]
I watch movies at 1.5 speed. I use adguard or ublock for ads.
Edge freezes as soon as the movie starts.
Firefox freezes 20 seconds after the movie starts and the sound lags.

**Reply by Dreamlinerm** [reply]
oh, then let me test with ublock enabled, I usually do not test with ublock enabled. Are you sure it's because of my extension?

**Reply by Dreamlinerm** [reply]
can you maybe go into private mode and only enable my extension to see if the problem still happens?

**Reply by Dreamlinerm** [reply]
can you maybe show me the problem over a discord call, this would probably be easiest since i don't have access to primevideo.com. https://discord.gg/5fBYathA7d

**Reply by Dreamlinerm** [reply]
you could also try to disable the adblock feature for prime video in the settings and see if there still is a problem, since ublock should block the ad and see if there still is an issue

**Reply by ocuonline** [reply]
I tried disabling all add-ons one by one. It causes this problem when it works with add-ons that automatically accelerate when you start movies. It works normally on its own.

Could you please set the speed that we set in the next update to remain at the same level globally? For example: When you start the movie, it should automatically play at 1.5 speed.

The accelerator add-ons that I use are below.
EDGE: https://microsoftedge.microsoft.com/addons/detail/super-video-speed-control/iplmmmmbamkaonmhjjoaomjbigkjgglf
Firefox: https://addons.mozilla.org/tr/firefox/addon/global-speed/?utm_content=addons-manager-reviews-link&utm_medium=firefox-browser&utm_source=firefox-browser
